PowerPointLabs/PowerPointLabs/FunctionalTestInterface.Impl/Controller/TimerLabController.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls.Primitives;
+
+using PowerPointLabs.ActionFramework.Common.Extension;
+using PowerPointLabs.TimerLab;
+using TestInterface;
+
+namespace PowerPointLabs.FunctionalTestInterface.Impl.Controller
+{
+    [Serializable]
+    class TimerLabController : MarshalByRefObject, ITimerLabController
+    {
+        private static ITimerLabController _instance = new TimerLabController();
+
+        public static ITimerLabController Instance { get { return _instance; } }
+
+        private TimerPane _pane;
+
+        private TimerLabController() {}
+
+        public void OpenPane()
+        {
+            UIThreadExecutor.Execute(() =>
+            {
+                FunctionalTestExtensions.GetRibbonUi().OnAction(
+                    new RibbonControl("TimerLabButton"));
+                _pane = FunctionalTestExtensions.GetTaskPane(
+                    typeof(TimerPane)).Control as TimerPane;
+            });
+        }
+
+        public void ClickCreateButton()
+        {
+            if (_pane != null)
+            {
+                UIThreadExecutor.Execute(() =>
+                {
+                    _pane.TimerPaneWPF.createButton.RaiseEvent(new RoutedEventArgs(ButtonBase.ClickEvent));
```

Comment:
Currently, all the other controllers are using the if (_pane != null) thing. To keep the code consistent, I think we should keep it as if (_pane != null) for now.

On another note, while I was looking through how to implement the null-conditional operators, it does not seem possible to implement it for stuff like line 49, where I did this:

_pane.TimerPaneWPF.DurationTextBox.Value = value;

Here I would have to check if pane is null before doing the assignment, right?